src/NServiceBus.AcceptanceTests/Core/Diagnostics/When_overriding_input_queue_name.cs

```diff
@@ -18,7 +18,7 @@ public async Task Should_use_custom_queue_names()
         Assert.Multiple(() =>
         {
             Assert.That(context.Done, Is.True);
-            Assert.That(context.InputQueue, Does.StartWith("OverriddenInputQueue"));
+            Assert.That(context.InputQueue, Does.StartWith("OverriddenInputQueue".ToLowerInvariant()));
```

Comment:
Obvious that this should not be changed like this but intentionally using `ToLower` here as the values in `context` can be different per transport.

@andreasohlund is that a correct assumption?